doc/source/dask-on-ray.rst

```diff
@@ -25,50 +25,73 @@ Here's an example:
 
    import ray
    from ray.util.dask import ray_dask_get
-   import dask.delayed
+   import dask.array as da
+   import dask.dataframe as dd
+   import numpy as np
+   import pandas as pd
    import time
 
    # Start Ray.
    # Tip: If you're connecting to an existing cluster, use ray.init(address="auto").
    ray.init()
 
+   d_arr = da.from_array(np.random.randint(0, 1000, size=(256, 256)))
 
-   @dask.delayed
-   def inc(x):
-       time.sleep(1)
-       return x + 1
+   # The Dask scheduler submits the underlying task graph to Ray.
+   d_arr.mean().compute(scheduler=ray_dask_get)
 
-   @dask.delayed
-   def add(x, y):
-       time.sleep(3)
-       return x + y
+   # Set the scheduler to ray_dask_get in your config so you don't have to specify it on
+   # each compute call.
+   dask.config.set(scheduler=ray_dask_get)
+
+   df = dd.from_pandas(pd.DataFrame(np.random.randint(0, 100, size=(1024, 2)), columns=["age", "grade"]))
+   df.groupby(["age"]).mean().compute()
```

Comment:
I think we should write more examples in a different section, but we can probably do that later.

Reply:
Also, can you write a note under this section that every Dask API should be available, but if users find any issue, create an issue to our github issue page?